docker/Dockerfile.wifi-connect

```diff
@@ -0,0 +1,32 @@
+FROM balenalib/BALENA_ARCH-python
```

Comment:
If possible, make use of just `-python-run` instead of `-python`. More details could be found [here](https://hub.docker.com/r/balenalib/aarch64-python) for instance, specifically on **_Image Variants_**.

Reply:
If possible, pin this to a specific tag like `3.7-buster`.

Reply:
The `-python-run` and `-python` images are of the same size, at least for the `3.7-buster` tag.